SoftwareGuide/Latex/DevelopmentGuidelines/CreateAModule.tex

```diff
@@ -739,6 +739,25 @@ \subsubsection{Wrapping Variables}
 
 \normalsize
 
+
+\subsubsection{Wrapping Enumerations}
+
+Enumeration classes need to be wrapped through the class they are declared in
+using the \code{itk\_wrap\_simple\_class}, e.g.:
+\small
+\begin{minted}
+itk_wrap_simple_class("itk::MathematicalMorphologyEnums")
+\end{minted}
+\normalsize
+
+which results in access to the class in Python as
+\code{itk.MathematicalMorphologyEnum}, its \code{Algorithm} enumeration class
+being accesible as \code{itk.MathematicalMorphologyEnums.Algorithm}, and its
+enum-list being accessed as
+\code{itk.MathematicalMorphologyEnums.Algorithm.BASIC},
+\code{itk.MathematicalMorphologyEnums.Algorithm.HISTO}, etc.
```

Comment:
@thewtex Have not tested any of these cases.

Reply:
@jhlegarreta close, `Algorithm.BASIC` -> `Algorithm_BASIC`, etc.

Reply:
Changed in 088a010.